src/renderer/components/settings/module.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="module" v-bind:class="{ loading: loading }">
-    <div class="dimmer">
-      <div class="large slow loader"></div>
+  <div class="module">
+    <div class="ui inverted dimmer" :class="{ active: loading }">
+      <div class="ui text loader">Loading</div>
     </div>
-    <div class="grid-container" v-if="!loading">
+    <div class="ui container form" v-if="!loading">
       <SettingsGroup :group="mainGroup"
```

Comment:
Don't render this group if it has no fields, it becomes an empty segment at the top of the page.